docs: updated airgap binaries and ova list DOC-925 (#1848)
spectrocloud/librarium: docs/docs-content/vertex/install-palette-vertex/airgap/supplemental-packs.md

| Kubernetes 1.25.9    | u-2004-0-k-1259-fips.ova  | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-1259-fips.ova`	|
| Kubernetes 1.25.10   | u-2004-0-k-12510-fips.ova | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-12510-fips.ova`	|
| Kubernetes 1.26.4    | u-2004-0-k-1264-fips.ova  | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-1264-fips.ova`	|
| Kubernetes 1.26.5    | u-2004-0-k-1265-fips.ova  | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-1265-fips.ova`	|
| Kubernetes 1.27.1    | u-2004-0-k-1271-fips.ova  | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-1271-fips.ova`	|
| Kubernetes 1.27.2    | u-2004-0-k-1272-fips.ova  | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-1272-fips.ova`	|
| RKE2 1.25.10          | u-2004-0-k-rke2-1250-fips.ova  | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-rke2-1250-fips.ova`	|
| RKE2 1.26.5          | u-2004-0-k-rke2-1265-fips.ova  | `https://vmwaregoldenimage-console.s3.us-east-2.amazonaws.com/u-2004-0-k-rke2-1265-fips.ova`	|

### Usage Instructions

> Use the provided download URL to download the OVA and upload it to your vSphere environment. The OVA must be uploaded to the `spectro-templates ` older VerteX can access it. One additional note about uploaded OVAs, the OVA name must be prefixed with `r_` to be recognized by VerteX. You can rename the OVA before you upload it or after you upload it to vSphere. 

Reviewer: ```suggestion
Use the provided URL to download the OVA and upload it to your vSphere environment. The OVA must be uploaded to `spectro-templates ` so older VerteX can access it. 

::: caution
The OVA name must be prefixed with `r_` so that VerteX recognizes it. You can rename the OVA either before or after you upload it to vSphere. 
```
:::

Reviewer: Missing word(s) in line 116 - "so older versions of VerteX recognize it"?

Author: I have no idea where older came from. Nice catch.

<Tabs>
<TabItem label="curl" value="curl">

> The following example shows how to download the `u-2004-0-k-1259-fips.ova` OVA and rename it to `r_u-2004-0-k-1259-fips.ova` by using `curl` and the `--output` flag. 

Reviewer: ```suggestion
The following example shows how to download the `u-2004-0-k-1259-fips.ova` OVA and rename it to `r_u-2004-0-k-1259-fips.ova` using `curl` and the `--output` flag. 
```
